server/applicationPattern/applicationPattern/rest/server/ResponseHeader.js

```diff
@@ -31,7 +29,6 @@ class ResponseHeader {
             this.xCorrelator = ResponseHeader.xCorrelatorGenerator();
         }
         this.execTime = ResponseHeader.executionTimeInMilliseconds(startTime);
-        this.backendTime = this.execTime;
```

Comment:
Kindly please let me know why the backendTime is getting removed from the ResponseHeader ?  Many thanks. 
Additionally, remove this change from this pull request , create a separate pull request for this change if necessary, as the context of the attached issue is not related to this change.